docs/src/main/sphinx/sql/merge.md

```diff
@@ -31,28 +31,38 @@ WHEN NOT MATCHED [ AND condition ]
 Conditionally update and/or delete rows of a table and/or insert new
 rows into a table.
 
-`MERGE` supports an arbitrary number of `WHEN` clauses with different
-`MATCHED` conditions, executing the `DELETE`, `UPDATE` or `INSERT`
-operation in the first `WHEN` clause selected by the `MATCHED`
-state and the match condition.
-
-For each source row, the `WHEN` clauses are processed in order.  Only
-the first first matching `WHEN` clause is executed and subsequent clauses
-are ignored.  A `MERGE_TARGET_ROW_MULTIPLE_MATCHES` exception is
+`MERGE`changes data in the `target_table` based on the contents of the
+`source_table`. The `search_condition` defines a condition, such as a relation
+from identical columns, to associate the source and target data.
+
+`MERGE` supports an arbitrary number of `WHEN` clauses. `MATCHED` conditions can
+execute `DELETE` or `UPDATE` operations on the target data, while `NOT MATCHED`
+conditions can add data from the source to the target table with `INSERT`.
+Additional conditions can narrow down the affected rows.
+
+For each source row, the `WHEN` clauses are processed in order. Only
+the first matching `WHEN` clause is executed and subsequent clauses
+are ignored. A `MERGE_TARGET_ROW_MULTIPLE_MATCHES` exception is
 raised when a single target table row matches more than one source row.
 
 If a source row is not matched by any `WHEN` clause and there is no
 `WHEN NOT MATCHED` clause, the source row is ignored.
 
 In `WHEN` clauses with `UPDATE` operations, the column value expressions
-can depend on any field of the target or the source.  In the `NOT MATCHED`
+can depend on any field of the target or the source. In the `NOT MATCHED`
 case, the `INSERT` expressions can depend on any field of the source.
 
+Typical usage of `MERGE` involves two identical tables in terms of structure,
```

Comment:
Added this now after talking to @electrum as well

Reply:
Rather than "two identical tables in terms of structure", maybe "two tables with similar structure"?

Reply:
Done